CONTRIBUTING.md

```diff
@@ -8,6 +8,42 @@ When you submit a pull request, a CLA bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+# Contributions that we appreciate
+
+## Submit a pull request that aligns with our goals and priorities
+
+ * Look for pinned issues
+ * Check issues under the labels `high priority` and `help wanted`
+ * New contributors may also look into issues labeled `good first issue`
+
+## Review a pull request
+
+We welcome reviews from contributors! Especially if this review happens before maintainers' review.
+To get an idea about our Code Review process, see:
+ * Ongoing reviews in [Pull Requests List](https://github.com/microsoft/STL/pulls)
+ * [Advice for Reviewing Pull Requests](https://github.com/microsoft/STL/wiki/Advice-for-Reviewing-Pull-Requests).
+ * [Code Review Videos](https://github.com/microsoft/STL/wiki/Code-Review-Videos!)
+
+# PR Checklist
+
+Before submitting a pull request, please ensure that:
+
+* Identifiers in product code changes are properly `_Ugly` as per
+  https://eel.is/c++draft/lex.name#3.1 or there are no product code changes.

```

Comment:
Do we often link to https://eel.is/c++draft in official documentation for the repo?  I have seen it cited in PR comments and issues on occasion, but since it's technically an unofficial source...
> (Generated on 2021-11-07 from the LaTeX sources by cxxdraft-htmlgen. This is not an ISO publication.
For historical versions of the document, see Tim Song's cppwp page.)
> Note: this is an early draft. It's known to be incomplet and incorrekt, and it has lots of bad formatting.

...I wonder if we should instead cite a specific working draft?

Reply:
I think we should go with [[lex.name]/3.1](https://eel.is/c++draft/lex.name#3.1)

Reply:
We use stable names in source comments and we do use draft references